packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -462,12 +462,13 @@ export class FocusZone extends React.Component<FocusZoneProps> implements IFocus
 
   /**
    * Handle global tab presses so that we can patch tabindexes on the fly.
+   * HEADS UP: This must not an arrow function in order to be referentially equal among instances for ref counting to work correctly!
    */
-  _onKeyDownCapture = (ev: KeyboardEvent) => {
+  _onKeyDownCapture(ev: KeyboardEvent) {
```

Comment:
I suggest to move out to be a function instead as it does not have any dependencies to `FocusZone` instance. In this case there is no need to have a dedicated comment.